core/server/common/src/main/java/alluxio/conf/ServerConfiguration.java

```diff
@@ -345,13 +346,13 @@ public static InstancedConfiguration global() {
    *
    * @param address the master address
    */
-  public static synchronized void loadClusterDefaultsIfNotLoaded(InetSocketAddress address)
+  public static synchronized void loadClusterDefaultsIfNotLoadedForWorker(InetSocketAddress address)
       throws AlluxioStatusException {
     if (sConf.getBoolean(PropertyKey.USER_CONF_CLUSTER_DEFAULT_ENABLED)
         && !sConf.clusterDefaultsLoaded()) {
       GetConfigurationPResponse response = ConfigurationUtils.loadConfiguration(address, sConf,
           false, true);
-      AlluxioConfiguration conf = ConfigurationUtils.getClusterConf(response, sConf);
+      AlluxioConfiguration conf = ConfigurationUtils.getClusterConf(response, sConf, Scope.WORKER);
```

Comment:
@calvinjia do we now also have client on worker (or job worker)? Do we need to respect cluster-default client-side setting on worker / job worker too?

Reply:
In my testing, there is one long-running client in leading master (not sure about the followers) and one long-running client on each worker. Those clients may be used for loading configuration from the leading master.